functions_framework.gemspec

```
@@ -46,6 +46,7 @@ version = ::FunctionsFramework::VERSION
 
   spec.required_ruby_version = ">= 3.1.0"
   spec.add_dependency "cloud_events", ">= 0.7.0", "< 2.a"
+  spec.add_dependency "logger", "~> 1.7"
```

I would consider removing the version restriction

```suggestion
  spec.add_dependency "logger"
```